mode/type/standalone/repository/provider/jdbc/pom.xml

```diff
@@ -61,5 +61,10 @@
             <artifactId>mysql-connector-j</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <scope>test</scope>
```

Comment:
I see from the dependency analyzer that the `shardingsphere-standalone-mode-repository-jdbc` module's dependency on `logback-classic` is already in `test` scope. Can you please help check it?

<img width="533" alt="image" src="https://github.com/user-attachments/assets/3aed0ec9-40ea-4ddb-9f76-a2d14f0fa369">

Reply:
- After I removed the declaration of `ch.qos.logback:logback-classic` here, I didn't see any logback dependencies. Without explicit declaration, the `mode/type/standalone/repository/provider/jdbc/src/test/resources/logback-test.xml` of this module will be useless. Where does this picture come from?
- ![image](https://github.com/user-attachments/assets/102c2525-787f-445b-8d69-4be2da99f724)

Reply:
My screenshot is from here

<img width="1643" alt="image" src="https://github.com/user-attachments/assets/51654d18-558a-4ed7-856a-2044c0bbfcc3">

Reply:
It looks like you are looking at the dependencies of the master branch. On the master branch, logback in test scope is pulled into all modules, as I pointed out in https://github.com/apache/shardingsphere/pull/32389#discussion_r1703174856.